Reason: plugin-statusnotifier: fix FTBFS with dbusmenu-qt
Source: https://github.com/lxde/lxqt-panel/commit/a28a27ceaec57d626d105065543792e486168df2
Upstream: yes, but reverted
--- lxqt-panel-0.10.0/plugin-statusnotifier/CMakeLists.txt.orig	2015-12-01 15:36:22.658313429 +0800
+++ lxqt-panel-0.10.0/plugin-statusnotifier/CMakeLists.txt	2015-12-01 15:36:54.758503583 +0800
@@ -2,6 +2,7 @@
 
 
 find_package(dbusmenu-qt5 REQUIRED)
+include_directories(${dbusmenu-qt5_INCLUDE_DIRS})
 
 set(HEADERS
     statusnotifier.h
